tests/testthat/test-ggMarginal.R

```diff
@@ -0,0 +1,25 @@
+# Wrap up the ggMarginal visual tests in a function runMarginalTests so that it's easy to test under multiple versions of ggplot2
+runMarginalTests <- function(ggplot2Version) {
+
+  context(ggplot2Version)
+  
+  test_that("ggMarginal can produce basic marginal plots" , {
```

Comment:
Sure, just having your testing framework in place would be a huge benefit, I won't be too opinionated on this :)